doc/releases/octopus.rst

```diff
@@ -588,6 +588,3779 @@ Upgrade compatibility notes
+* cephadm: bootstrap: warn on fqdn hostname (`pr#33042 <https://github.com/ceph/ceph/pull/33042>`_, Sage Weil)
+* cephadm: check for both chrony service names (`pr#33369 <https://github.com/ceph/ceph/pull/33369>`_, Sage Weil)
+* cephadm: check for both ntp.service and ntpd.service (`pr#32302 <https://github.com/ceph/ceph/pull/32302>`_, Sage Weil)
+* cephadm: clean up the systemd unit and ceph-crash shutdown behavior (`pr#32685 <https://github.com/ceph/ceph/pull/32685>`_, Sage Weil)
+* cephadm: correct ipv6 support in port open detection (`pr#32286 <https://github.com/ceph/ceph/pull/32286>`_, Paul Cuzner)
+* cephadm: create /var/run/ceph/$fsid as needed (`pr#32390 <https://github.com/ceph/ceph/pull/32390>`_, Sage Weil)
+* cephadm: disable node-exporter cpu/memory limits for the time being (`pr#33133 <https://github.com/ceph/ceph/pull/33133>`_, Sage Weil)
+* cephadm: drop sha256: prefix on container id (`pr#32300 <https://github.com/ceph/ceph/pull/32300>`_, Sage Weil)
+* cephadm: error out on filestore OSDs (`pr#33395 <https://github.com/ceph/ceph/pull/33395>`_, Sage Weil)
+* cephadm: fix adoption safety check (`pr#33445 <https://github.com/ceph/ceph/pull/33445>`_, Sage Weil)
+* cephadm: fix ceph version probe (`pr#33136 <https://github.com/ceph/ceph/pull/33136>`_, Sage Weil)
+* cephadm: fix container cleanup (`pr#32282 <https://github.com/ceph/ceph/pull/32282>`_, Sage Weil)
+* cephadm: fix datetime regexp to capture at most 6 digits (`pr#33932 <https://github.com/ceph/ceph/pull/33932>`_, Michael Fritch)
+* cephadm: fix deploy crash when no `args.fsid` (`pr#33248 <https://github.com/ceph/ceph/pull/33248>`_, Michael Fritch)
+* cephadm: fix error handing in `command_check_host()` (`pr#33048 <https://github.com/ceph/ceph/pull/33048>`_, Guillaume Abrioux)
+* cephadm: fix failure when getting keyring for deploying daemons (`pr#33679 <https://github.com/ceph/ceph/pull/33679>`_, Kiefer Chang)
+* cephadm: fix help message for bootstrap --mgr-id (`pr#32640 <https://github.com/ceph/ceph/pull/32640>`_, Sage Weil)
+* cephadm: fix inspect-image (`pr#33109 <https://github.com/ceph/ceph/pull/33109>`_, Sage Weil)
+* cephadm: fix logging defaults (`pr#32641 <https://github.com/ceph/ceph/pull/32641>`_, Sage Weil)
+* cephadm: fix name argument parsing during image check for non-ceph components (`pr#33114 <https://github.com/ceph/ceph/pull/33114>`_, Daniel-Pivonka)
+* cephadm: Fix Py3 ConfigParser deprecation warnings (`pr#32218 <https://github.com/ceph/ceph/pull/32218>`_, Michael Fritch)
+* cephadm: fix tox DeprecationWarning (`pr#32753 <https://github.com/ceph/ceph/pull/32753>`_, Michael Fritch)
+* cephadm: fix v1/v2 ip/addrv handling; explicitly check bind to ip:port (`pr#32392 <https://github.com/ceph/ceph/pull/32392>`_, Sage Weil)
+* cephadm: fix `alertmanager not implemented yet` (`pr#33694 <https://github.com/ceph/ceph/pull/33694>`_, Patrick Sei
```

Comment:
Can we write "Keithley" instead of ALL-CAPS?

Reply:
fixed, i did a find and replace, but starting point was after this :)